src/ophyd_async/core/device_save_loader.py

```diff
@@ -151,11 +135,7 @@ def save_to_yaml(phases: Sequence[Dict[str, Any]], save_path: str) -> None:
     """
 
     yaml.add_representer(np.ndarray, ndarray_representer, Dumper=yaml.Dumper)
-
-    yaml.add_representer(ca_float, ca_dbr_representer, Dumper=yaml.Dumper)
-    yaml.add_representer(ca_int, ca_dbr_representer, Dumper=yaml.Dumper)
-    yaml.add_representer(ca_str, ca_dbr_representer, Dumper=yaml.Dumper)
-    yaml.add_representer(ca_array, ca_dbr_representer, Dumper=yaml.Dumper)
+    yaml.add_representer(ca_array, ndarray_representer, Dumper=yaml.Dumper)
```

Comment:
...
That should get rid of all the `ca_array`s and turn them into regular numpy arrays, so we can lose this line

Reply:
Can we now lose this line?